google/resumable_media/requests/download.py

```diff
@@ -19,6 +19,9 @@
 from google.resumable_media.requests import _helpers
 
 
+_SINGLE_GET_CHUNK_SIZE = 8192
```

Comment:
SG, thanks! It's funny we ended up with the same constant. I was just winging it, but figured 8K was better than 1K!